.github/workflows/pattern-metrics.yaml

```diff
@@ -0,0 +1,120 @@
+name: Monthly issue metrics
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '3 2 1 * *'
```

Comment:
> “At 02:03 on day-of-month 1.”

Alright, why not :)

Reply:
haha. I picked a random time on the first day of the month. People tend to do things on the hour and jobs get backed up so I like to have mine at odd times to avoid queuing. :)